src/index.ejs

```diff
@@ -76,7 +76,11 @@
             {
               type:'welcome',
               options:{
-                title:'Open your new bank account'
+                title:'Open your new bank account',
+                descriptions: [
+                  'To open a bank account, we will need to verify your identity.',
```

Comment:
I don't think this is documented in the steps options. We should probably add this to the readme unless we don't want to enforce the customisation of the body in the welcome screen.